docs/reference/announcements/870.md

```diff
@@ -18,18 +18,18 @@ Important changes and updates for the Camunda 8.7 release are summarized below.
   - [Fetch RPA resource API](#)
   - [deployResourceAPI for RPA](#) -->
 
-- [Web Modeler Deploy diagram change](#web-modeler-deploy-diagram-change)
-- [Southwest Asia for SaaS region](#southeast-asia-region-for-saas-customers-saas)
+- [Web Modeler Deploy diagram change Self-Managed](#web-modeler-deploy-diagram-change-self-managed)
+- [Southeast Asia region for SaaS customers SaaS](#southeast-asia-region-for-saas-customers-saas)
```

Comment:
It looks like the links here pulled in the badge text - they could probably be reworded from the anchor text.

```suggestion
- [Self-Managed Web Modeler deploy diagram change](#web-modeler-deploy-diagram-change-self-managed)
- [Southeast Asia region for SaaS customers](#southeast-asia-region-for-saas-customers-saas)
```